docs/paictl/deployment.md

```diff
@@ -68,39 +70,35 @@ deploy-rules:
 
-
 - ${operator}-script: Corresponding operator will call the script in the yaml file. And the script is wrote by component developers. So the way, how to handle the operator to the component, is designed and programed by component developers.
 
 - deploy-rules: Specify which role(label) of machine will run the service. By setting this filed the service will be scheduled onto the specific node with the configured label.
 
-
 #### Service schedule policy
+
 The last section mentioned that by setting "deploy-rules" in `service.yaml`, the service will be scheduled onto the specific node with the configured label. This section will introduce the service schedule policy.
 
-To schedule the service onto the specific node, all you only need to do is configuring the `service.yaml` "deploy-rules" field, then PAI will automatically add [NodeAffinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity) to the service's deployment yaml file when start the service, such as [grafana.yaml.template](https://github.com/Microsoft/pai/blob/master/src/grafana/deploy/grafana.yaml.template) and complete the schedule process. 
+To schedule the service onto the specific node, all you only need to do is configuring the `service.yaml` "deploy-rules" field, then PAI will automatically add [NodeAffinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity) to the service's deployment yaml file when start the service, such as [grafana.yaml.template](https://github.com/Microsoft/pai/blob/master/src/grafana/deploy/grafana.yaml.template) and complete the schedule process.
 
-"deploy-rules" is an array composed of several component rules. We support multiple rules. A item of rule includes two parts: operator and label. Currently support two kinds of operators: "in" and "notin", the operater decide the service will be deploy on the node "has" or "not has" the label. The label definition of machines is in [cluster-configuration.yaml](https://github.com/Microsoft/pai/blob/master/examples/cluster-configuration/cluster-configuration.yaml#L93). Machines will be labeled automatically when you deploy PAI for the first time. Label config in `service.yaml` should keep the consistent with it in `cluster-configuration.yaml`,  otherwise it will throw an error.
+"deploy-rules" is an array composed of several component rules. We support multiple rules. A item of rule includes two parts: operator and label. Currently support two kinds of operators: "in" and "notin", the operater decide the service will be deploy on the node "has" or "not has" the label. The label definition of machines is in [layout.yaml](https://github.com/Microsoft/pai/blob/master/examples/cluster-configuration/layout.yaml#L49). Machines will be labeled automatically when you deploy PAI for the first time. Label config in `service.yaml` should keep the consistent with it in `layout.yaml`,  otherwise it will throw an error.
```

Comment:
relative links like below is better for versionning, and line number is not reliable.
examples/cluster-configuration/layout.yaml

Reply:
Agree! Fixed it.